src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/importorder/Example5.java

```diff
@@ -7,13 +7,16 @@
   </module>
 </module>
 */
+//non-compiled with javac: Compilable with Java11
+package com.puppycrawl.tools.checkstyle.checks.imports.importorder;
 
 // xdoc section -- start
 import static java.io.File.createTempFile;
 import java.lang.Math.sqrt;
 
-import javax.swing.JComponent; // violation, extra separation in import group
+import javax.swing.JComponent; // violation '...separation in import group...'
```

Comment:
// violation, 'Extra separation in import group'

Reply:
> // violation, 'Extra separation in import group'

Done.